aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -47,6 +47,19 @@ module aptos_token::token {
     const EWITHDRAW_ZERO: u64 = 22;
     const ENOT_TRACKING_SUPPLY: u64 = 23;
     const ENFT_NOT_SPLITABLE: u64 = 24;
+    const ECOLLECTION_NAME_TOO_LONG: u64 = 25;
+    const ENFT_NAME_TOO_LONG: u64 = 26;
+    const EURI_TOO_LONG: u64 = 27;
+
+    //
+    // Constants
+    //
+
+    const MAX_COLLECTION_NAME_LENGTH: u64 = 32;
```

Comment:
Have you looked at any collections to tell whether they'll all fit in this amount of characters?